server/controllers/blog.js

```diff
@@ -0,0 +1,75 @@
+const { Op } = require('sequelize');
+const { Blog } = require('../models');
+
+//  getting all the blog post limited to first 10
+
+exports.getAllBlog = (req, res, next) => {
+  const count = 10;
```

Comment:
I think the limit is a good idea, but we'll need a way to fetch pages as well if a band has more than 10 blog posts.